internal/pkg/template/templates/environment/cf.yml

```diff
@@ -69,7 +69,7 @@ Resources:
       Configuration:
         ExecuteCommandConfiguration:
           Logging: DEFAULT
-{{- if .Telemetry }}
+{{- if and .Telemetry .Telemetry.EnableContainerInsights}}
```

Comment:
Does it work if `.Telemetry.EnableContainerInsights` is `&false`?

Reply:
good catch!